src/utils.ts

```diff
@@ -1,9 +1,51 @@
+import { readdirSync } from 'fs'
+import { inspect } from 'util'
+import { groupBy, uniq } from 'lodash'
+
+const HOLOCHAIN_DATABASE_DIRECTORY = process.env.NODE_ENV === 'test'
+  ? './'
```

Comment:
This seems sketchy. Why do we want to use the current directory in tests? In an e2e test it seems like we would want to get the actual database directory.